libcore/FileUtils.vala

```diff
@@ -919,6 +919,142 @@ namespace PF.FileUtils {
         return result;
     }
 
+    public const string COPY_TAG = N_("(copy");
+    public const string LINK_TAG = N_("(link");
+    public const string CLOSING_COPY_LINK_TAG = N_(")");
```

Comment:
In an ideal world, to make translators life easier, it would be something like :
```suggestion
    public const string COPY_TAG = N_("(copy %d)");
    public const string LINK_TAG = N_("(link %d)");
```
and use `_(COPY_TAG).split("%d",2)` to get the previous `COPY_TAG` and `CLOSING_COPY_LINK_TAG`.
I'm unsure that assuming the same closing tag would work for RTL languages.

Reply:
I like the direction though ;)

Reply:
Yes RTl languages are problematic.  If you use `split` then wouldn't the closing tag be the __first__ item not the last?  Maybe to make translation easier there should be an opening tag as well as a closing tag, with translator comments provided to explain what function these characters serve.

Reply:
So you need an even more complicated string matching function with:
`%s (copy %d)`
and do some `split("%s",2)` and look for each part for a `%d`

Reply:
So, am I correct in thinking strings  of text in RTL languages are stored "normally" (i.e. opening character first) and just displayed RTL?  I believe that in an RTL language an opening parenthesis character "(" is actually displayed as ")"?  Very confusing. :confused:

Reply:
According to Google Translate: `File (Copy 1)` is translated to `ملف (نسخ 1)` in arabic

Reply:
Just pushed a commit with a separate opening tag and translator comments.  This works OK using RTL in an English environment at least (using GTK_DEBUG=interactive). I guess there are some languages that do not use "(" but I do not know which.

Reply:
Hold on, one of the tests fails.

Reply:
Fixed the test